src/signals/incident-management/containers/IncidentOverviewPage/components/PageHeader/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { createStructuredSelector } from 'reselect';
+import styled from 'styled-components';
+import * as types from 'shared/types';
+
+import PageHeader from 'components/PageHeader';
+import { makeSelectIncidentsCount } from 'signals/incident-management/containers/IncidentOverviewPage/selectors';
+import { makeSelectActiveFilter } from 'signals/incident-management/selectors';
+import { makeSelectQuery } from 'models/search/selectors';
+
+import Refresh from '../../../../../../shared/images/icon-refresh.svg';
+
+const RefreshIcon = styled(Refresh).attrs({
+  height: 18,
+})`
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 15px;
+`;
+
+export const PageHeaderContainerComponent = ({
+  incidentsCount,
+  filter,
+  children,
+  query,
+}) => {
+  let title = filter.name || 'Meldingen';
+  const hasCount = !!incidentsCount && isNaN(Number(incidentsCount)) === false; // eslint-disable-line no-restricted-globals
```

Comment:
I like the explicit `=== false`. Probably easier to read than `!isNaN`

Personally also prefer to user `Boolean()` over `!!` for readability